content/en/docs/usage/kubectl-plugin.md

```diff
@@ -13,24 +13,29 @@ While the kubectl plugin is supported, it is recommended to use
 [cmctl](../cmctl/) as this enables a better experience via tab auto-completion.
 
 ## Installation
+
 You need the `kubectl-cert-manager.tar.gz` file for the platform you're using, these can be found on our [GitHub releases page](https://github.com/jetstack/cert-manager/releases).
 In order to use the kubectl plugin you need its binary to be accessible under the name `kubectl-cert_manager` in your `$PATH`.
 
-### macOS/Linux
+### Linux/MacOS
```

Comment:
This will  change the anchors links in the docs and cause broken links for external sites that link directly to this documentation:
 * /docs/usage/kubectl-plugin/#macos-linux VS
 * /docs/usage/kubectl-plugin/#linux-macos

Please revert this change and then I think this can be merged.

Reply:
Done